hlint.cabal

```diff
@@ -67,14 +67,15 @@ library
         extra >= 1.6.6,
         refact >= 0.3,
         aeson >= 1.1.2.0,
-        filepattern >= 0.1.1
+        filepattern >= 0.1.1,
+        ghc-lib-parser-ex == 8.8.2
```

Comment:
Can I not be 8.8.* anymore? If not, I'm going to have to rerelease HLint with 8.8.3

Reply:
We can leave `ghc-lib-parser` at 8.8.*.

I need to be more specific for `ghc-lib-parser-ex` for now because otherwise I could release new versions which `hlint` would then pick up automatically perhaps breaking it until its adapted.

My thinking then was to pin them both for the moment and take responsibility for caring for them during this time in the face of upgrades.

Reply:
Hmm, that's going to be problematic in the long run - I wonder if ghc-lib-parser-ex should be `8.8` as the GHC version (with the promise that it works with the whole 8.8 series), and then the next digit should be the ghc-lib-parser-ex breaking version (starting at 0), and the final one the minor revision (starting at 0 and increasing), then i'd pin to `8.8.0.*`. Thoughts?

Reply:
Like `ghc-lib`, I also intend to keep `ghc-lib-parser-ex` working against GHC head builds (e.g. `ghc-lib-parser-0.20200121`). What to do?! 😁

Reply:
> Can I not be 8.8.* anymore? If not, I'm going to have to rerelease HLint with 8.8.3

May I suggest we go with this approach for now? Careful management of the compatibility of these dependencies is guaranteed and there's no way we're going to miss an upgrade of `ghc-lib-parser` 😉 That said, still open to adopting a better approach.